index.html

```diff
@@ -69,10 +107,23 @@ <h6><b>SIGN IN</b> <span class="subscribe">Subscribe ></span></h6>
               </i></li>
           </ul>
         </div>
-    
       </nav>
     </div>
   </header>
+
+  <script>
+    window.onscroll = function() {myFunction()}; 
+    var header = document.getElementById("stickyHeader");
+    var sticky = header.offsetTop;
+    function myFunction() {
+      if (window.pageYOffset > sticky) {
+     header.classList.add("sticky");
+  } else {
+    header.classList.remove("sticky");
+  }
+} 
+  </script>
```

Comment:
Can you figure out a way to do this without javascript? it is possible.

Reply:
I forgot to validate the HTML after the last changes, will do that right now.

About that, I could do something similar without JS but it wouldn't look like the original one, do you want me to change that or is it optional?